apps/web/app/sign-up/page.tsx

```diff
@@ -0,0 +1,13 @@
+import { AuthCard } from "@/components/auth-card";
+import AuthCardWrapper from "@/components/auth-card-wrapper";
+import React from "react";
+
```

Comment:
Don't forget to add metadata for new pages

https://nextjs.org/docs/app/getting-started/metadata-and-og-images